README.md

```diff
@@ -255,7 +254,8 @@ Table of Contents
 
   * https://www.simplybuilt.com - SimplyBuilt offers free website building and hosting for open source projects (http://www.simplybuilt.com/explore/free-websites-for-open-source-projects). Simple alternative to GitHub Pages.
   * http://www.devport.co - Turn GitHub projects, Apps, and websites into a personal developer portfolio.
-  * https://www.netlify.com - Builds, deploy and hosts static site or app, free for 100 MB data and 1 GB bandwith.
+  * https://www.netlify.com - Builds, deploy and hosts static site or app, free for 100 MB data and 1 GB bandwidth.
+  * https://pages.github.com - Host websites directly from your GitHub repo. 
```

Comment:
this  is a great PR thanks, I'd prefer we dont list every feature of every app in different catogories, will soon make a website for this and then support tags or similar.  Till then I'd prefer we dont add this one

thanks